scripts/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -4,8 +4,6 @@
 # ${CLUSTER_API_REPO}/cmd/clusterctl/hack/create-local-repository.py
 # For CAPI release v0.3.12, the folder v0.3.8 is created
 
-export UPGRADED_K8S_VERSION="v1.21.1"
```

Comment:
why are we removing this?

Reply:
In scale-in && node reuse tests we need the same kind of var in order to upgrade k8s version in the test. Currently `UPGRADED_K8S_VERSION` is only used in upgrade tests, and lives inside the upgrade specific files, so it was removed/moved to upper root file(common.sh) so any test can access it.